### PR TITLE
Add ProjectDetails Wizard Page 3/c

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Adds the initial first page for ProjectDetails Page.

This is not complete, but a starting point

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1970** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970" target="_blank">Graphite</a> 
    * **PR #1971** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1971" target="_blank">Graphite</a> 
      * **PR #1972** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1972" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1972?utm_source=stack-comment).

File: src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { FormImageUploader } from 'components/inputs/FormImageUploader'
+import { CreateCollapse } from '../../CreateCollapse'
+
+// TODO: Do we want to set validators on these?
+export const ProjectDetailsPage: React.FC = () => {
+  return (
+    <Form name="projectDetails" colon={false} layout="vertical">
+      <Space direction="vertical" size="large" style={{ width: '100%' }}>
+        <Form.Item name="projectName" label={t`Project Name`}>
+          <Input placeholder={t`Enter project name`} />
```
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx:12]: **suggestion** I didn't point it out in design but IMO we should avoid all placeholders. https://www.nngroup.com/articles/form-design-placeholders/

File: src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            placeholder={t`Enter project description`}
+          />
+        </Form.Item>
+        <Form.Item name="logo" label={t`Logo`}>
+          <FormImageUploader text={t`Upload`} />
+        </Form.Item>
+        <CreateCollapse>
+          <CreateCollapse.Panel key={0} header={t`Project Links`}>
+            {/* Adding paddingBottom is a bit of a hack, but horizontal gutters not working */}
+            <Row gutter={32} style={{ paddingBottom: '32px' }}>
```
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx:26]: **suggestion**: 

```suggestion
            <Row gutter={32} style={{ paddingBottom: '2rem' }}>
```

**Review by tomquirk — APPROVED** [review]
preapproving, 2 comments!

File: src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx (hunk 2 above)
Comment (wraeth-eth) [reply src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx:26]: Ah thank you - probably have missed a few of these

**Pull request merged**